MetaMask/metamask-mobile
# fix: PPOMController update to handle storage crash

## **Description**
PPOMController update to handle storage crash

## **Related issues**

Ref: MetaMask/MetaMask-planning#1701

## **Manual testing steps**

1. Enable blockaid and go to test dapp
2. Trigger malicious transaction
3. It should sow error 

## **Screenshots/Recordings**
NA

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
